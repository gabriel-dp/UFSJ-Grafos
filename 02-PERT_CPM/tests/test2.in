1 2 10
1 3 7
2 4 5
4 5 3
3 6 0
5 6 2
6 7 1
7 8 14