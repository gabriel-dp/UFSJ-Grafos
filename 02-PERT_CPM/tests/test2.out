| ## | Min | Max | Slack |
|  1 |   0 |   0 |     0 |
|  2 |  10 |  10 |     0 |
|  3 |   7 |  20 |    13 |
|  4 |  15 |  15 |     0 |
|  5 |  18 |  18 |     0 |
|  6 |  20 |  20 |     0 |
|  7 |  21 |  21 |     0 |
|  8 |  35 |  35 |     0 |

Critical path:
( 1) -> ( 2) -> ( 4) -> ( 5) -> ( 6) -> ( 7) -> ( 8)
